Source/Core/VideoCommon/VertexLoader_Color.h

```diff
@@ -4,14 +4,42 @@
 #pragma once
 
 #include "Common/CommonTypes.h"
+#include "Common/EnumMap.h"
+#include "VideoCommon/CPMemory.h"
 #include "VideoCommon/VertexLoader.h"
 
-enum class VertexComponentFormat;
-enum class ColorFormat;
-
 class VertexLoader_Color
 {
+  template <typename T, auto last_member>
+  using EnumMap = typename Common::EnumMap<T, last_member>;
+
+  using SizeTable = EnumMap<EnumMap<u32, ColorFormat::RGBA8888>, VertexComponentFormat::Index16>;
```

Comment:
Assuming that it's allowed (which I *think* is the case), I'd rather these were put after the `private:` instead of at the top of the class (in both cases, they're `private`, as `class` members default to `private` until an access modifier is specified (`struct` members default to `public`), but this is definitely an implementation detail and not the first thing that should be seen when looking at the class).

Reply:
Will do. I did it that way because I was trying to match how it's done in EnumMap.